### DISCUSSION TITLE
Unable to Use "pybind11.get_cmake_dir()"

### DISCUSSION THREAD

**Discussion by ramen-newdals** [pybind/pybind11#4394] [Q&A] [answered]
Hello All,

The title pretty much says it all I have installed pybind 11 with the following command
`pip install pybind11`
And the package is sucesfully installed, but when i try to call 
`pybind11.get_cmake_dir()`
i get the following error message:
`AttributeError: module 'pybind11' has no attribute 'get_cmake_dir`
Using pip freeze i am running pybind11==2.2.4.
Any assistance in understanding why this error pops up is greatly appreciated.

**Comment by Skylion007** [comment, marked as answer]
That's a positively ancient version, we are on 2.10 now. Just update the newest version that is supported by your Python version.
